packages/headless/src/topic.ts

```diff
@@ -40,6 +40,18 @@ export function useTopics(query: ITopic.FindTopicsApiQuery) {
   return usePaginate(findTopics, [QueryKey.FindTopic, query]);
 }
 
+export function useUserTopics() {
+  const atlas = useAtlas();
+  const findTopics = useCallback(async () => {
+    return await atlas.topic.findUserTopics();
+  }, [atlas.topic]);
+
+  return useQuery({
+    queryFn: findTopics,
+    queryKey: [QueryKey.FindTopic],
```

Comment:
For the first glance, the term `FindTopics` will make us think that you are trying to get all the topics we have in the backend. We can make it more specific by naming it `FindUserTopics`

```suggestion
    queryKey: [QueryKey.FindUserTopics],
```